sphinx_book_theme/theme.conf

```diff
@@ -17,6 +17,7 @@ navbar_footer_text =
 extra_navbar = Theme by the <a href="https://ebp.jupyterbook.org">Executable Book Project</a>
 extra_footer =
 use_issues_button = False
-use_repository_button = False
+use_download_button = False
```

Comment:
hmm - `use_repository_button` should remain `False` (the default that we currently have), and `use_download_button` should be `True` (also our current default behavior)

Can you also add documentation for this feature to https://github.com/executablebooks/sphinx-book-theme/blob/master/docs/configure.md ?

Reply:
Sorry, I obviously edited the wrong line 🤦🏻‍♂️. I hope I got it right this time...

I'll proceed to the documentation and be back.